### PR TITLE
Calender Highlight

### PR DESCRIPTION
**Pull request by Tibinsunny**
Hey,
I've just added a feature to highlight dates with notes 😋

Cheers ,

### CODE REVIEW

**Review by A5H1Q — COMMENTED** [review]
Does it work on striked-out dates (past) ?

**Comment by Tibinsunny** [comment]
Not sure.Notes Tweak wont work with past dates any ways . So highlighting past wont be necessary right ?

File: src/index.js [hunk 1]
```diff
@@ -31,12 +31,20 @@ const App = () => {
...
+ {
+preHighlight.map((data,index) => {
+    highlights.push(new Date(data.id))
+})
+ }
  // Calender Configs see: @natscale/react-calender
  const [calOptions] = useState({
   useDarkMode: false,
   startOfWeek: 0,
   disablePast: true,
```
Comment (A5H1Q) [inline src/index.js:46]: We could make it work by setting this as false (disablePast flag) or removing that line, but then the visual UX of past days in the widget would be gone..

**Review by A5H1Q — APPROVED** [review]
Gr8 work my man

File: src/index.js (hunk 1 above)
Comment (Tibinsunny) [reply src/index.js:46]: That that should work fine 😀

Comment (A5H1Q) [reply src/index.js:46]: Also forgot to mention that there's currently no way to delete a note. And the freshly highlighted date won't be seen on the widget until the page is refreshed

Comment (Tibinsunny) [reply src/index.js:46]: I'll soon add those an make another pr 😜

**Pull request merged**